src/Symfony/Component/Notifier/Bridge/Telegram/TelegramOptions.php

```diff
@@ -109,4 +109,19 @@ public function edit(int $messageId): static
 
         return $this;
     }
+
+    /**
+     * @return $this
+     */
+    public function answerCallbackQuery(string $callbackQueryId, bool $showAlert = false, int $cacheTime = 0): static
+    {
+        $this->options['callback_query_id'] = $callbackQueryId;
+        $this->options['show_alert'] = $showAlert;
+
+        if ($cacheTime > 0) {
+            $this->options['cache_time'] = $cacheTime;
+        }
```

Comment:
Is this check necessary or can we just sent cache_time = 0 all the time?

Reply:
Should we allow negative value be passed? API itself does not seem to complain if negative value is sent, though it does not make sense to me

Reply:
I've extended TelegramOptionsTest with cases when cache_time is not set due to it being zero or negative integer